cli/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 
 _Released 1/30/2024_
 
+**Bugfixes:**
+
+- Fixed an issue with capturing assets for Test Replay when service workers are registered in Cypress support files. This issue would cause styles to not render properly in Test Replay. Fixes [#28747](https://github.com/cypress-io/cypress/issues/28747).
```

Comment:
We would need to move this entry in the changelog once it is ready to go.
```suggestion
```

Reply:
What should the next release date be?